cmd/viewer/tests/tests_clone.go

```diff
@@ -426,14 +426,18 @@ func (src *StructWithContainers) Clone() *StructWithContainers {
 	dst := new(StructWithContainers)
 	*dst = *src
 	dst.CloneableContainer = *src.CloneableContainer.Clone()
-	dst.ClonableGenericContainer = *src.ClonableGenericContainer.Clone()
+	dst.CloneableGenericContainer = *src.CloneableGenericContainer.Clone()
+	dst.CloneableMap = *src.CloneableMap.Clone()
```

Comment:
wouldn't this panic if `CloneableMap` is nil?

Reply:
`ClonableMap` cannot be nil, as the field is a value rather than a pointer:
```
CloneableMap MapContainer[int, *StructWithPtrs]
```
We do handle nilable fields, however. For example,
https://github.com/tailscale/tailscale/blob/801064190f7b164ee6952fe154e3b8048b4282a7/cmd/viewer/tests/tests_clone.go#L303C2-L305C3
But we do this regardless of the field types.